### ISSUE TITLE
Only synced lyrics without animated background.

### ISSUE THREAD

**Issue by thenoobtester** [harmonoid/harmonoid#320]
**Describe the new feature you'd like**

Thank you for this great and beautiful application. As much as I love the animated background could it be possible to deactivate it but keep the synced lyrics as it is pretty mean on my  CPU, the animated background that is.

Thank you.

**Comment by alexmercerind** [comment]
Hi @thenoobtester,
You can press the buttons in the bottom-right to go to non-animated-background screen, where dominant color from your album art will be used. It'll be less heavy on your processor.

Also, you can add custom image from the settings (even `1px`x`1px`) with your favourite color, which won't be painted again & again (and it won't cramp your CPU).

See those arrow buttons in the bottom right:

![](https://user-images.githubusercontent.com/28951144/187735756-615774ac-d267-4989-8188-aa1866cc46d4.png)

**Comment by alexmercerind** [comment]
Close the issue if you feel it has been answered.

<hr></hr>

I had plans to implement music-sensitive visualizers which will be less heavy for CPU & look better, but I'm not on my limits & personally I feel satisfied with the experience.
If community wants it & wishes to sponsor the project, I will have more time for this project.


https://user-images.githubusercontent.com/28951144/187736998-87ac7cb2-0e28-43ef-b31e-b8851bb9d15e.mp4

**Comment by alexmercerind** [comment]
I feel it being answered.

**Comment by thenoobtester** [comment]
Sorry for the late response. You answered my question, thank you :)

**Comment by thenoobtester** [comment]
Quick update, I think I was wrong by assuming it was the animated backgrounds that caused the CPU spike for me, even though when no animated background is used, a slight decrease can be observed. After playing a piece of music with no lyrics I noticed a sharp drop in CPU (with both animated/ not animated background). So I played a song with lyrics and the CPU usage increased again, I disabled/hid the lyrics but kept the background, then used a non-animated background but the CPU usage was still the same. So my conclusion is that it's the synced lyrics that caused the high CPU usage because when they are hidden the CPU usage remains high whereas when we are dealing with a piece of music with no lyrics at all the CPU usage is correct for a music player.

Whether you decide to investigate or not I just thought I would let you know since I noticed.
